ansible/roles/pre-ansible/defaults/main.yml

```diff
@@ -6,4 +6,6 @@ bin_dir: /usr/bin
 # reside.
 bootstrap_script_dir: /opt/bin
 pypy_version: 2.4.0
-pypy_base_url: https://bitbucket.org/pypy/pypy/downloads
+
+pypy_download_url_base: "https://bitbucket.org/pypy/pypy/downloads"
+pypy_download_url: "{ pypy_download_url_base }}/pypy-{ pypy_version }}-linux64.tar.bz2"}}
```

Comment:
@rutsky Good catch, I did not test this in a clear environment, give me a few and I'll run it in vagrant. 

Also caught a mistake on the end of the url with the }}